django/template/loader.py

```diff
@@ -4,25 +4,20 @@
 
```

Comment:
The tests for the changes in this file are in ``tests/template_loader/tests.py``.